docs/book/src/cronjob-tutorial/testdata/project/cmd/main.go

```diff
@@ -75,13 +75,16 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var secureMetrics bool
 	var enableHTTP2 bool
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8443", "The address the metric endpoint binds to. "+
-		"Use 0 in order to disable the metrics server")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", "0", "The address the metrics endpoint binds to. "+
+		"Use :8443 for HTTPS or :8080 for HTTP, or leave as 0 to disable the metrics service.")
```

Comment:
If we change the default here from 0, we cannot really disable the metrics when we comment the metrics patch.
So, my intention here was keep the option to disable + clarify how that works